### ISSUE TITLE
Google translate accessibility issues

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1788]
The Translate link/button in the primary menu doesn't work correctly. Some of the issues affect all users, some affect keyboard-only users, and some affect screen reader users.

1. All Users: when my screen is small so that the primary menu is at the bottom of the window with a hamburger menu showing, pressing the globe icon with the link to open the Google Translate window doesn't work at all. Nothing happens even when I use a mouse to press it.

2. Screen reader users: this Translate feature is not available at all to these users which is a big failure for accessibility. The main problem here is that the aria-hidden="true" attribute is added to the HTML for the link (but only the Font Awesome icon should have aria-hidden="true"):
<a href="#" id="navbar_translate" title="Translate" data-proud-navbar="" class="btn navbar-btn translate-button proud-navbar-processed" aria-hidden="true" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
<i aria-hidden="true" class="fa fa-fw fa-globe"></i>
Translate
</a>

This prevents screen reader users from even hearing there is a Translate link. The screen reader announces "Action toolbar tool bar section". Announcing a tool bar section that has no links or buttons in it is meaningless. If I remove aria-hidden from the link, then the screen reader announces "Action toolbar toolbar Translate link collapsed submenu" which is correct and allows these users to activate the link to expand or collapse the submenu and be able to translate the page. You can't assume that screen reader users don't want translation capability.

3. Keyboard-only users (includes screen reader users): I can tab to the globe icon and press Enter to activate it which opens a submenu that has a drop down to select a language and a link to Google Translate, but I can't easily close this submenu with the keyboard unless I think to Shift-tab backwards to the globe icon and press Enter on it again. The normal user interaction with this kind of submenu would be that once focus is moved out of the submenu then it would close. This is what happens if I can use a mouse and click anywhere outside of the submenu. Note that the Escape key does work to close it, but if I'm using a screen reader then I wouldn't see the open submenu that stays even after focus has moved away from it.

**Comment by aschmoe** [comment]
@kevindherman took care of 1 + 2... 3 however unfortunately is out of our control because that is just the google widget.

**Comment by kevindherman** [comment]
@aschmoe thank you!
